Helper/Config.php

```diff
@@ -16,6 +16,8 @@ class Config extends \Magento\Framework\App\Helper\AbstractHelper
 
     const STOREKEEPER_TOKEN = 'storekeeper_general/general/storekeeper_token';
 
+    const IS_DEBUG_LOGS = 'storekeeper_general/general/debug_logs';
+
```

Comment:
Add expected stock settings to the Readme.md
![image](https://user-images.githubusercontent.com/1073778/223654273-96bea4c6-145c-4f21-8243-407c83875ddc.png)